src/storage/build.ts

```diff
@@ -20,7 +20,6 @@ export function canStale(value: CachedStorageValue): boolean {
     Header.ETag in headers ||
     Header.LastModified in headers ||
     Header.XAxiosCacheEtag in headers ||
-    Header.XAxiosCacheStaleIfError in headers ||
```

Comment:
Seems unrelated 🤔 Also is it used somewhere else?

Reply:
This was an internal header, that I think should not be checked here.
I might be wrong though

Reply:
Keep this statement for now, unrelated to this PR. And i'll move from internal headers to a meta property on storage values soon :) Or is it causing any bugs?

Reply:
It is, because it prevents stale data from expiring correctly.

Reply:
Alright, i'm going to move the others anyway any time soon :)